src/pages/user_profile/ControlledAccessGrants.jsx

```diff
@@ -11,12 +11,6 @@ const headCells = [
     disablePadding: false,
     label: 'DAR Code',
   },
-  {
-    id: 'approvalDate',
-    numeric: false,
-    disablePadding: false,
-    label: 'Approval Date',
-  },
```

Comment:
Removing this in anticipation of https://broadworkbench.atlassian.net/browse/DT-1878?focusedCommentId=100448